### DISCUSSION TITLE
Many web sites can not be reached

### DISCUSSION THREAD

**Discussion by damienmarsic** [strongswan/strongswan#559] [Show and tell]
**System (please complete the following information):**
 - OS: Arch Linux
 - Kernel version (if applicable): 5.13.12-arch1-1
 - strongSwan version(s): Linux strongSwan U5.9.3/K5.13.12-arch1-1
 - Tested/confirmed with the latest version: yes

**Describe the bug**
I use Strongswan to use NordVPN services. Many websites could not be accessed. It took me 2 months to find a solution, which I will share with you. Web sites which could not be accessed when the VPN was on: gmail, gmx, duolingo, cerego, linkedin, scmp (and probably many others, those are just what I personally use). After discussions on the archlinux forum, someone finally helped me with a solution: lower the MTU. I tried different values, the highest value beyond which the websites were no longer accessible was 1490. Concretely, I created a rules file in /etc/udev/rules.d/ (which I named 10-network.rules) with the following content:
ACTION=="add", SUBSYSTEM=="net",  ATTR{mtu}="1490"
By just doing this, all websites can be accessed without any problem.

**Comment by Thermi** [comment]
That's a bad solution.
Use `charon.plugins.kernel-netlink.mtu` and `.mss` in a `strongswan.conf` file instead. That way the MTU of only the routes over the VPN are lowered.
